liveviz/src/styles/graph.css

```diff
@@ -1,15 +1,34 @@
-div.graphSection{
+div.graphSection {
     position: absolute;
     top: 70px;
     bottom: 0;
     left: 0;
     right: 0;
 }
 
-div.graphRow{
+div.graphRow {
     height: 50%;
 }
 
+#graph-params > div.col-md-12 {
+    width: 100%;
```

Comment:
I think it's better to let bootstrap calculate the width.
I don't want the width to be hard-coded.